app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.csc.lesson3">
+    package="ppzh.ru.translator">
```

Comment:
Common package name rules is domain zone - company - project, i.e. `ru.ppzh.translator` in your case